integrations/zendesk/src/definitions/actions.ts

```diff
@@ -0,0 +1,107 @@
+import z from 'zod'
+import { ticketSchema, userSchema } from './schemas'
+
+const createTicket = {
+  title: 'Create Ticket',
+  description: 'Creates a new ticket in Zendesk',
+  input: {
+    schema: z.object({
+      subject: z.string().describe('Subject for the ticket'),
+      comment: z.string().describe('Comment for the ticket'),
+      requesterName: z.string().describe('Requester name'),
+      requesterEmail: z.string().describe('Requester email'),
+      __conversationId: z.string().describe('Internal: Conversation ID to bind the ticket to'),
```

Comment:
**question:** why is there a double underscore here ? I don't think I like that

Reply:
This is temporary until we have a way to mark some properties as "hidden" (we use them internally, but don't want the user to see/change them in the ui)